### PR TITLE
[MSCONFIG] [LOGOFF] Add Latvian (lv-LV) translation

### PR DESCRIPTION
**Pull request by Coool**
msconfig, logoff lv-LV.rc (Latvian) by Coool (github.com/Coool)

### CODE REVIEW

**Review by binarymaster — CHANGES_REQUESTED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email coool@mail.lv
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email coool@mail.lv
```

Also please set up your GitHub account accordingly, so maintainers can use Squash & Merge button without getting the author field modified back to nickname by GitHub, it's done in two steps:

1. Set your full name in [GitHub profile settings](https://github.com/settings/profile) (it should match your `user.name` setting)
1. Uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails)

Regards,

File: base/applications/msconfig/lang/lv-LV.rc [hunk 1]
```diff
@@ -0,0 +1,173 @@
+/* Latvian language resource file (github.com/Coool, 2022-11-02) */
```
Comment (binarymaster) [inline base/applications/msconfig/lang/lv-LV.rc:1]:
```suggestion
/*
 * PROJECT:     ReactOS System Configuration Tool
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Latvian resource file
 * TRANSLATOR:  Copyright 2022 Your Name <your@email.com>
 */
```

File: base/applications/msconfig/lang/lv-LV.rc [hunk 2]
```diff
@@ -1,4 +1,9 @@
-/* Latvian language resource file (github.com/Coool, 2022-11-02) */
+/*
+* PROJECT:     ReactOS System Configuration Tool
+* LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+* PURPOSE:     Latvian resource file
+* TRANSLATOR:  Copyright 2022 Coool (github.com/Coool) <coool@mail.lv>
```
Comment (binarymaster) [inline base/applications/msconfig/lang/lv-LV.rc:5]: This has to be your full legal name, see [CONTRIBUTING](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations)

**Review by binarymaster — CHANGES_REQUESTED** [review]

File: base/applications/msconfig/lang/lv-LV.rc (hunk 2 above)
Comment (Coool) [reply base/applications/msconfig/lang/lv-LV.rc:5]: Hmm, I don't want to share my real name.

Comment (binarymaster) [reply base/applications/msconfig/lang/lv-LV.rc:5]: Sorry, but your contribution wouldn't be accepted then. As you can see, we all are following the rules: https://git.reactos.org/?p=reactos.git;a=shortlog

Comment (Coool) [reply base/applications/msconfig/lang/lv-LV.rc:5]: Что же делать?

Comment (binarymaster) [reply base/applications/msconfig/lang/lv-LV.rc:5]: You have two options, either use real full name, or close Pull Request.

(also please, we are speaking English here to understand each other, we are international community)

**Comment by Coool** [comment]
I will think about it. It's up to ReactOS staff to decide to merge.

**Comment by leandrofriedrich** [comment]
Who's gonna tell him...

**Pull request closed without merging**